contactform.php

```diff
@@ -441,6 +441,12 @@ public function sendMessage()
                 [],
                 'Shop.Notifications.Error'
             );
+        } elseif (strlen(Tools::getValue('from')) > 255) {
```

Comment:
What happens for previous version of PrestaShop with a shorter field length? 
I guess it should take the value from Contact::$definition if available.

Reply:
If PS < 1.7.5 with this fix:

without this PR:
=255: error 500 
>255: error 500
<255: send email

with this PR:
=255: error 500 //what is the chance?
>255: display error message
<255: send email

If PS = 1.7.5 with this fix:
with this PR:
=255: display error message
>255: display error message
<255: send email

Reply:
@MathiasReker I follow @Quetzacoalt91. Please use the definition of fhe field.